contentcuration/contentcuration/frontend/channelEdit/views/ImportFromChannels/BrowsingCard.vue

```diff
@@ -230,4 +239,8 @@
     }
   }
 
+  .card-header {
```

Comment:
This class doesn't seem to be applied, so the header `font-size` is ending up as 20px instead of 18px

Reply:
Nice catch! Removed it